Flutter/pdf-viewer/select-deselect-annotations.md

```diff
@@ -0,0 +1,128 @@
+
+This section will go through the various functions available in the `SfPdfViewer` for selecting and deselecting annotations in a PDF document.
+
+## Select an annotation
+
+You can select an annotation by simply tapping on the annotation using touch or mouse. When the annotation is selected, the selection border (selector) appears, indicating that the annotation is selected. 
+
+### Select an annotation programmatically
+
+You can select an annotation programmatically by providing the annotation instance as the parameter to the `selectAnnotation` method of PdfViewerController. The annotation instance can be found in the `getAnnotations` method of the PdfViewerController. The following example explains how to select the first annotation in the annotation collection.
+
+{% tabs %}
+{% highlight dart hl_lines="8" %}
+
+void selectFirstAnnotation() {
+  // Get the list of annotations in the PDF document.
+  List<Annotation> annotations = _pdfViewerController.getAnnotations();
+  if (annotations.isNotEmpty) {
+    // Get the first annotation from the PDF document.
+    Annotation firstAnnotation = annotations.first;
+    // Select the first annotation in the PDF document.
+    _pdfViewerController.selectAnnotation(firstAnnotation);
+  }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Annotation selected callback
+
+The callback provided to `onAnnotationSelected` property is triggered when an annotation is selected interactively or programmatically. The selected annotation instance can be obtained from the callback details. The following code snippet explains how to use this callback.
+
+{% tabs %}
+{% highlight dart hl_lines="7 8 9 10" %}
+
+@override
+Widget build(BuildContext context) {
+  return MaterialApp(
+    home: Scaffold(
+      body: SfPdfViewer.network(
+        'https://cdn.syncfusion.com/content/PDFViewer/flutter-succinctly.pdf',
+        onAnnotationSelected: (Annotation annotation) {
+          // Instance of the selected annotation.
+          Annotation selectedAnnotation = annotation;
+        },
+      ),
+    ),
+  );
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customizing selector appearance
+
+The `annotationSettings` property of `PdfViewerController` allows you to customize the default selector color. The following example explains how to customize the selector color for locked and unlocked annotations.
+
+{% tabs %}
+{% highlight dart %}
+
+void customizeSelectorAppearance() {
+  PdfAnnotationSettings annotationSettings =
+      _pdfViewerController.annotationSettings;
+  // For unlocked annotations.
+  annotationSettings.selector.color = Colors.blue;
+  // For locked annotations.
+  annotationSettings.selector.lockedColor = Colors.grey;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Deselect an annotation
+
+You can deselect an annotation by simply tapping outside of its bounds using touch or mouse. When the annotation is deselected, the selection border (selector) disappears, indicating that the annotation is deselected.
```

Comment:
You can deselect an annotation by tapping outside its bounds using a touch or mouse. When the annotation is deselected, the selection border (selector) disappears, indicating that the annotation is deselected.